src/test/airtable/test_airtable_round_winners.test.js

```diff
@@ -47,19 +47,19 @@ beforeEach(async function () {
       }
     },
     {
-      id: 'proposal_1',
+      id: 'proposal_2',
       fields: {
         'Project Name': 'Oblong Apple',
         'USD Requested': 20000,
         'Voted Yes': 1000,
-        'Voted No': 0
+        'Voted No': 1000
```

Comment:
Hi @claudiaHash, was this added to address Edge Case (2)?

> "What happens when a project get's 0 votes?"

If so, I would have expected my comment further below to address the notFundedResults.length to be 2
`should.equal(notFundedResults.length, 1)`

Thank you for updating the tests & validations!

Reply:
Hi! thanks for checking and pointing that out, I'll make the needed check and push the changes